web-library-build/src/index.ts

```diff
@@ -34,30 +34,33 @@ const sourceMatch: string[] = [
 ];
 
 // Define default task groups.
-let buildTasks: IExecutable = task('build', serial(preCopy, sass, parallel(tslint, typescript, text), postCopy));
-let bundleTasks: IExecutable = task('bundle', serial(buildTasks, webpack));
-const postProcessSourceMaps: PostProcessSourceMaps = new PostProcessSourceMaps();
-const validateShrinkwrapTask: ValidateShrinkwrapTask = new ValidateShrinkwrapTask();
-const generateShrinkwrapTask: GenerateShrinkwrapTask = new GenerateShrinkwrapTask();
+export const buildTasks: IExecutable = task('build', serial(preCopy, sass, parallel(tslint, typescript, text), postCopy));
+export const bundleTasks: IExecutable = task('bundle', serial(buildTasks, webpack));
+export const compileTsTasks: IExecutable = parallel(typescript, text);
+export const testTasks: IExecutable = serial(sass, compileTsTasks, karma);
+export const defaultBuildBundleAndTest: IExecutable = serial(bundleTasks, karma);
```

Comment:
Just wanted to make it more explicit what default does, renamed it to `buildBundleAndTestDefaultTask` to make it more apparent.